test/Furlongs.jl

```diff
@@ -0,0 +1,110 @@
+module Furlongs
```

Comment:
Can you add a comment with an explanation and a link to the original file this is copied from?